database/driver.js

```diff
@@ -226,6 +228,8 @@ function discernCollectionFromType(type) {
   
   switch (type) {
     case "Project":
+    case "Canvas":
```

Comment:
I see what's happening here, but we will bork soem things if we have a data object that looks like a Layer  or Line and we just save it to the "Projects" collection.